src/pages/home/ReportScreen.tsx

```diff
@@ -766,93 +832,103 @@ function ReportScreen({route, navigation}: ReportScreenProps) {
     const shouldDisplayMoneyRequestActionsList = isMoneyRequestOrInvoiceReport && shouldDisplayReportTableView(report, reportTransactions ?? []);
 
     return (
-        <ActionListContext.Provider value={actionListValue}>
-            <ReactionListWrapper>
-                <ScreenWrapper
-                    navigation={navigation}
-                    style={screenWrapperStyle}
-                    shouldEnableKeyboardAvoidingView={isTopMostReportId || isInNarrowPaneModal}
-                    testID={`report-screen-${reportID}`}
-                >
-                    <FullPageNotFoundView
-                        shouldShow={shouldShowNotFoundPage}
-                        subtitleKey={shouldShowNotFoundLinkedAction ? '' : 'notFound.noAccess'}
-                        subtitleStyle={[styles.textSupporting]}
-                        shouldShowBackButton={shouldUseNarrowLayout}
-                        onBackButtonPress={shouldShowNotFoundLinkedAction ? navigateToEndOfReport : Navigation.goBack}
-                        shouldShowLink={shouldShowNotFoundLinkedAction}
-                        linkKey="notFound.noAccess"
-                        onLinkPress={navigateToEndOfReport}
-                        shouldDisplaySearchRouter
+        <KeyboardGestureArea
+            style={styles.flexGrow1}
+            offset={90}
+            interpolator="ios"
+            textInputNativeID="composer"
+        >
```

Comment:
Just wanted to highlight, that usage of this component will automatically enable interactive dismissal on Android (so we have to test that platform carefully as well).

Also if you discover any kind of bugs/undesired behavior, don't hesitate to remove it and test the implementation on iOS without this component. If `KeyboardGestureArea` produces any bugs, then let me know and I'll try to fix them!

(posting it here because `KeyboardGestureArea` relatively new component on iOS and may have some bugs 😅)